src/activities/RegisterToken.ts

```diff
@@ -0,0 +1,53 @@
+import type { IActivityHandler } from "@geocortex/workflow/runtime";
+import IdentityManager from "@arcgis/core/identity/IdentityManager";
+
+interface RegisterTokenInputs {
+    /**
+     * @description The URL of the server that the access token is valid for. For example: https://www.arcgis.com/sharing/rest; https://www.example.com/portal/sharing/rest; https://www.example.com/arcgis/rest/services.
+     * @required
+     */
+    server: string;
+
+    /**
+     * @description The access token.
+     * @required
+     */
+    token: string;
+
+    /**
+     * @description The token expiration time specified as number of milliseconds since 1 January 1970 00:00:00 UTC.
```

Comment:
Should we describe the default behaviour if you don't specify a value here?

Reply:
I was thinking about that too, but the behaviour is not documented. It is very likely that means that it never expires, but I'd rather not make a claim we don't know to be true.

Reply:
Right, and if they ever change the implementation we've got to update our inline help to match. I think what we have is adequate as we already link to Esri's docs for `IdentityManager.registerToken()`.